draft-ietf-jsonpath-base.md

```diff
@@ -251,8 +251,7 @@ Children (of a node):
 
 Descendants (of a node):
 : The children of the node, together with the children of its children, and so forth
-  recursively. More formally, the descendants relation between nodes is the transitive
-  closure of the children relation.
+  recursively.
```

Comment:
I'd probably leave the sentence in and just use quotes for "descendants" and "children" before relation.

Reply:
Done, thanks. The mathematician in me is quietly pleased.